### PR TITLE
stripped name of invalid characters for edge

### PR DESCRIPTION
**Pull request by botsteve**
added the function that replaced all forbidden characters with '_'. This is the behaviour of the submit action on forms for Edge.

Fix for #580 and #584

### CODE REVIEW

**Review by StanciuMarian — APPROVED** [review]

**Review by eligrey — CHANGES_REQUESTED** [review]
If Edge needs these changes then these changes should only be applied for the versions of Edge that need this.

**Review by botsteve — COMMENTED** [review]
finish adding required changes

**Review by botsteve — COMMENTED** [review]
finish implementing changes

File: src/FileSaver.js [hunk 1]
```diff
@@ -16,6 +16,14 @@ var _global = typeof window === 'object' && window.window === window
   ? global
   : this
 
+function stripNameOfInvalidChars(name) {
```
Comment (botsteve) [inline src/FileSaver.js:19]: Modified

Comment (botsteve) [reply src/FileSaver.js:19]: added specific edge version for fix

Comment (botsteve) [reply src/FileSaver.js:19]: @eligrey when you have time to review

**Comment by botsteve** [comment]
Bump @eligrey @jimmywarting

**Pull request closed without merging**